packages/node_modules/@webex/plugin-meetings/README.md

```diff
@@ -51,6 +51,20 @@ webex.meetings.unregister()
 
 #### Creating a basic meeting
 
+##### Via hydra room id
+
+```javascript
+let hydraRoomId = `TheForceHydraRoomId`;
```

Comment:
Can we use the sample hydra id here from `index.html`? 
```suggestion
const roomId = `Y2lzY29zcGFyazovL3VzL1JPT00vNWZhMWUzODAtZTkzZS0xMWU5LTgyZTEtOGRmYTg5ZTgzMjJm `;
```

Reply:
will do